app/src/main/java/org/oppia/app/topic/train/TopicTrainViewModel.kt

```diff
@@ -0,0 +1,24 @@
+package org.oppia.app.topic.train
+
+import android.view.View
+import androidx.databinding.ObservableField
+import androidx.lifecycle.ViewModel
+import org.oppia.app.fragment.FragmentScope
+import javax.inject.Inject
+
+/** [ViewModel] for showing skills in train fragment. */
+@FragmentScope
+class TopicTrainViewModel @Inject constructor(
+  private val topicTrainFragmentPresenter: TopicTrainFragmentPresenter
+) : ViewModel() {
+
+  var isSubmitButtonActive = ObservableField<Boolean>(false)
+
+  fun selectedSkillList(selectedSkillList: ArrayList<String>) {
```

Comment:
Method names ought to be verbs or verb phrases. Two possible alternate namings: ``updateSelectedSkillList`` or ``notifySelectedSkillList``. This helps clarify to callers what's happening when they call this method.

Reply:
Changed function name to `notifySelectedSkillList`